src/ocean/util/log/Config.d

```diff
@@ -447,8 +453,11 @@ public void configureLoggers ( Source = ConfigParser, FileLayout = LayoutDate,
         m_config = an instance of the MetaConfig class
         file_appender = delegate which returns appender instances to write to
                         a file
+        makeLayout      = A delegate that returns a `Layout` instance from
+                          a name, or throw on error.
+                          By default, wraps `ocean.util.log.Config.newLayout`
         use_insert_appender = true if the InsertConsole appender should be used
```

Comment:
This is one reason I don't like `/// ditto` generally -- it's usually an over-simplification. (There must be _some_ difference between the two things that's worth explaining, otherwise why have them both?)